instrumentation-tests/src/main/res/raw/reroute_response_dc_very_short.json

```diff
@@ -1 +1 @@
_typical":23.337,"duration_typical":11.154,"duration":7.663,"distance":89,"driving_side":"right","weight":19.138,"mode":"driving","geometry":"wb}diA``t|qCuj@E_E?"},{"voiceInstructions":[{"ssmlAnnouncement":"<speak><amazon:effect name=\"drc\"><prosody rate=\"1.08\">You have arrived at your destination.</prosody></amazon:effect></speak>","announcement":"You have arrived at your destination.","distanceAlongGeometry":79.167}],"intersections":[{"entry":[true,false],"in":1,"bearings":[91,180],"duration":6.405,"turn_weight":9,"turn_duration":4.315,"mapbox_streets_v8":{"class":"primary"},"is_urban":true,"admin_index":0,"out":0,"weight":11.561,"geometry_index":30,"location":[-77.03195,38.895447]},{"entry":[true,false],"in":1,"bearings":[90,271],"duration":5.117,"turn_duration":0.007,"mapbox_streets_v8":{"class":"primary"},"is_urban":true,"admin_index":0,"out":0,"weight":6.259,"geometry_index":31,"location":[-77.031744,38.895445]},{"bearings":[90,270],"entry":[true,false],"in":1,"mapbox_streets_v8":{"class":"primary"},"is_urban":true,"admin_index":0,"out":0,"geometry_index":32,"location":[-77.031237,38.895445]}],"bannerInstructions":[{"primary":{"components":[{"type":"text","text":"You will arrive at your destination"}],"type":"arrive","modifier":"straight","text":"You will arrive at your destination"},"distanceAlongGeometry":93},{"primary":{"components":[{"type":"text","text":"You have arrived at your destination"}],"type":"arrive","modifier":"straight","text":"You have arrived at your destination"},"distanceAlongGeometry":79.167}],"speedLimitUnit":"mph","maneuver":{"type":"turn","instruction":"Turn right onto Pennsylvania Avenue Northwest.","modifier":"right","bearing_after":91,"bearing_before":0,"location":[-77.03195,38.895447]},"speedLimitSign":"mutcd","name":"Pennsylvania Avenue Northwest","weight_typical":22.23,"duration_typical":15.122,"duration":15.122,"distance":93,"driving_side":"right","weight":22.23,"mode":"driving","geometry":"mt~diAz_t|qCB{K?u^?oU"},{"voiceInstructions":[],"intersections":[{"bearings":[270],"entry":[true],"in":0,"admin_index":0,"geometry_index":33,"location":[-77.030877,38.895445]}],"bannerInstructions":[],"speedLimitUnit":"mph","maneuver":{"type":"arrive","instruction":"You have arrived at your destination.","bearing_after":0,"bearing_before":90,"location":[-77.030877,38.895445]},"speedLimitSign":"mutcd","name":"Pennsylvania Avenue Northwest","weight_typical":0,"duration_typical":0,"duration":0,"distance":0,"driving_side":"right","weight":0,"mode":"driving","geometry":"it~diAx|q|qC??"}],"distance":957.042,"summary":"F Street Northwest, 13th Street Northwest"}],"geometry":"adaeiAz_t|qCcN?mP?wE??}H@_p@DyjA?[?uIlE@t}@_@zEJxEnCtU?|FAq@nEo@lGQ|COdH@zI@rYjCv@vCn@bE?zMHj\\RGvDHpv@SpIuj@E_E?B{K?u^?oU","voiceLocale":"en-US"}],"waypoints":[{"distance":3.551,"name":"14th Street Northwest","location":[-77.03195,38.896721]},{"distance":4.203,"name":"Pennsylvania Avenue Northwest","location":[-77.030877,38.895445]}],"code":"Ok","uuid":""}
```

Comment:
what did you change in the response? just updated with fresh response from Direction API, didn't you? 🤔

Reply:
Just added the `uuid` at some point. Not really needed anymore, I can revert.